compiler/rustc_metadata/src/dependency_format.rs

```diff
@@ -402,19 +402,34 @@ fn verify_ok(tcx: TyCtxt<'_>, list: &[Linkage]) {
             let cnum = CrateNum::new(i + 1);
             let found_strategy = tcx.panic_strategy(cnum);
             let is_compiler_builtins = tcx.is_compiler_builtins(cnum);
-            if is_compiler_builtins || desired_strategy == found_strategy {
+            if is_compiler_builtins {
```

Comment:
The variable could be inlined, and the `let found_strategy` moved lower down (to match the new code more).